tests/framework/utils/s3_operations.go

```diff
@@ -128,3 +128,14 @@ func (h *S3Helper) DeleteObjectInBucket(bucketname string, key string) (bool, er
 	}
 	return true, nil
 }
+
+//IsBucketPresent function returns true if a bucket is present and false if it's not present
+func (h *S3Helper) IsBucketPresent(bucketname string) bool {
+	_, err := h.s3client.HeadBucket(&s3.HeadBucketInput{
+		Bucket: aws.String(bucketname),
+	})
+	if err != nil && strings.Contains(err.Error(), "NotFound") {
```

Comment:
you'll need to return `(bool, error)` so that you can detect other error cases besides notFound. Otherwise, an error will show up as an existing bucket